### PR TITLE
homework - week2 completed

### PR DESCRIPTION
**Pull request by priyankapandey13**
Week 2 homework

### CODE REVIEW

File: week2/materials/homework/assignments.md [hunk 1]
```diff
@@ -4,51 +4,73 @@
...
 
 **Assignment 6:**
 
 Figure out and write down the price per month of storing 51TB on S3.
 
-Price: `?`
+Price: 
+`$0.023/GB for first 50TB = 1.15 for 50GB`
+`$0.022/GB for next 450TB = 0.022 for 1GB` 
+`$1.172*1000 = Total 1172$  for 51TB`
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md:24]: This is correct :)

File: week2/materials/homework/assignments.md [hunk 2]
```diff
@@ -4,51 +4,73 @@
...
 
-Total size of website: `?`
+Total size of website: `less then 51TB`
 
 **Assignment 7:**
 
 Write down the main cost factors for S3
 
-brief description: `?`
+brief description: `Storage Region : The region where you store your data. there are 21 different regions where it’s possible to store data. Prices are different for each storage region. `
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md:32]: Nice description 👍🏻

File: week2/materials/homework/assignments.md [hunk 3]
```diff
@@ -4,51 +4,73 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `s3://priyankas-good-greens/`
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md]: I am missing full URL to access it

File: week2/materials/homework/assignments.md [hunk 4]
```diff
@@ -4,51 +4,73 @@
...
+    `Data retrievals : Retrieving data from a Standard S3 storage volume is free of charge. However, if you place data in an Infrequent or Glacier storage class, charges apply when you retrieve it (and for requesting it, as outlined above). Pricing for data retrieval is based on the volume of data retrieved, but can vary depending on whether an expedited retrieval or bulk retrieval from an archive storage tier is required.`
 
 **Assignment 8:**
 
 The file `week2/assignments/products.json` needs to be uploaded Inside the bucket `hyf-serverless-course-week-2`. Upload the file through the CLI and write down the command needed.
 
-docs: 
+docs: `aws s3 cp D:\ServerLess\week2\materials\homework\products.json s3://hyf-serverless-course-week-2/`
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md:43]: 👍🏻

File: week2/materials/homework/assignments.md [hunk 5]
```diff
@@ -4,51 +4,73 @@
...
 
-bucket url: `?`
+bucket url: `s3://priyankas-good-greens/`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `CloudFront `
```
Comment (paulinadubas) [inline week2/materials/homework/assignments.md:14]: It would be nice if some explanation would be added.

**Review by paulinadubas — CHANGES_REQUESTED** [review]

File: week2/materials/homework/assignments.md (hunk 3 above)
Comment (priyankapandey13) [reply week2/materials/homework/assignments.md]: Thanks, Done 👍🏻

File: week2/materials/homework/assignments.md (hunk 5 above)
Comment (priyankapandey13) [reply week2/materials/homework/assignments.md:14]: Done 👍🏻 🙂

**Pull request merged**